Gemfile

```diff
@@ -0,0 +1,4 @@
+source 'https://rubygems.org'
```

Comment:
This is certainly possible, but I'm afraid that the invocation of the acceptance tests would then become the following:

```
bundle install && bundle exec rake -f spec/Rakefile beaker
```

In response to this, I have structured this invocation into a BASH script "accept.sh" (which can then, in turn, be invoked by users).